gm_dotnet_managed/Tests/GetUserTypeOnTypeMismatch.cs

```diff
@@ -0,0 +1,70 @@
+﻿using GmodNET.API;
```

Comment:
Doesn't seem related to documentation

Reply:
This test was added to ensure that documented behavior for GetUserType in fact holds.

Reply:
Oh ok